source/common/http/http1/codec_impl.cc

```diff
@@ -444,19 +445,23 @@ ConnectionImpl::ConnectionImpl(Network::Connection& connection, Stats::Scope& st
       enable_trailers_(enable_trailers),
       reject_unsupported_transfer_encodings_(Runtime::runtimeFeatureEnabled(
           "envoy.reloadable_features.reject_unsupported_transfer_encodings")),
-      output_buffer_([&]() -> void { this->onBelowLowWatermark(); },
-                     [&]() -> void { this->onAboveHighWatermark(); }),
+      dispatching_(false), output_buffer_([&]() -> void { this->onBelowLowWatermark(); },
```

Comment:
I'm unfamiliar with this code, but I think the reason why it was written like that was because client/server would need to capture a disjoint set of things. Could pass both active request/pending response in, I can try to

Reply:
oh, sorry. I misread the diff and thought you had added this. I'm not crazy about it but it's pre-existing so lgtm.